classes/checkout/CheckoutSession.php

```diff
@@ -196,4 +196,15 @@ public function getCheckoutURL()
     {
         return $this->context->link->getPageLink('order');
     }
+
+    public function setSeparatePackagesAllowed($separatePackagesAllowed)
+    {
+        $this->context->cart->allow_seperated_package = $separatePackagesAllowed;
```

Comment:
```suggestion
        $this->context->cart->allow_seperated_package = (bool) $separatePackagesAllowed;
```
If we do not force the type to bool, maybe we should cast it here to make sure we don't mess the cart 👀

Reply:
Also typo again 😅 
```suggestion
        $this->context->cart->allow_separated_package = $separatePackagesAllowed;
```